geode-core/src/main/java/org/apache/geode/internal/net/SocketCloser.java

```diff
@@ -41,6 +44,7 @@
  * This max threads can be configured using the "p2p.ASYNC_CLOSE_POOL_MAX_THREADS" system property.
  */
 public class SocketCloser {
+  private static final Logger logger = LogService.getLogger();
```

Comment:
no need to modify SocketCloser.java in this pr

Reply:
removed this unneeded log line